File: pinyougou-manager-web/src/main/resources/static/js/service/sellerService.js
//服务层
app.service('sellerService',function($http){

    this.findAllBrand=function () {
        return $http.get("../seller/findAllSeller")
    }

    this.findPageSeller=function (page,size,searchEntity) {
        return $http.post("../seller/findPageSeller?page="+page+"&size="+size,searchEntity)
    }

    this.findOne=function (id) {
        return $http.get("../seller/findOne?sellerId="+id)
    }

    this.del=function (ids) {
        return $http.post("../seller/delSeller",ids)
    }

    this.add=function (searchEntity) {
        return $http.post("../seller/addSeller",searchEntity)
    }

    this.update=function (searchEntity) {
        return $http.post("../seller/updateSeller",searchEntity)
    }

    this.updateStatus=function (id,status) {
        return $http.get("../seller/updateStatus?sellerId="+id+"&status="+status)
    }
});

File: interface/src/main/java/com/kgc/pinyougou/service/SellerService.java
package com.kgc.pinyougou.service;

import com.kgc.pinyougou.domain.Seller;
import com.kgc.pinyougou.utils.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SellerService {

	/**
	 * @Author Admin
	 * @description:  //TODO
	 * @param:
	 * @param pageNum   分页页数
	 * @param pageSize  分页记录数
	 * @return: com.kgc.pinyougou.utils.PageResult
	 * @time: 2020/4/25 22:11
	 */
	PageResult findAllSeller(Integer pageNum, Integer pageSize, Seller seller);

	Integer insertSeller(Seller seller);

	Seller findOne(String id);

	Integer updateSeller(Seller seller);

	Integer updateStatus(String sellerId,String status);

	Integer deleteSeller(String [] ids);


}
